nexus/proposals/api.py

```diff
@@ -0,0 +1,35 @@
+from django.utils import timezone
+
+# Third Party Stuff
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework import mixins
+from rest_framework.permissions import IsAuthenticated
```

Comment:
No need to put `IsAuthenticated` as it is applied as [policy](https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy) in the [settings.py](https://github.com/CuriousLearner/nexus/blob/cddebf331eaab01ecf30f3eb421a9068410d5b15/settings/common.py#L90)